DigitalLearningSolutions.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -23,6 +24,22 @@ public static class HtmlHelperExtensions
                 : string.Empty;
         }
 
+        public static string IsSelectedTab(

```

Comment:
Fair.
Don't think the selectedCssClass param is necessary - currently it's never used is it?
The css for selected tabs is specified once, in layout.scss, so the class name be the same for the whole site. It would be good to have "selected" extracted as a constant in this file though.